com.zsmartsystems.zigbee.dongle.cc2531/src/main/java/com/zsmartsystems/zigbee/dongle/cc2531/ZigBeeDongleTiCc2531.java

```diff
@@ -419,8 +450,8 @@ private int createEndPoint(int endpointId, int profileId) {
         logger.trace("Registering a new endpoint {} for profile {}", endpointId, profileId);
 
         AF_REGISTER_SRSP result;
-        result = networkManager.sendAFRegister(
-                new AF_REGISTER(endpointId, profileId, (short) 0, (byte) 0, new int[] {}, new int[] {}));
+        result = networkManager.sendAFRegister(new AF_REGISTER(endpointId, profileId, (short) 0, (byte) 0,
+                this.supportedInputClusters, this.supportedOutputClusters));
```

Comment:
I don't think we need to have ```this``` here as there are no local vars with the same name.